checker/src/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -109,6 +121,14 @@ protected TreeAnnotator createTreeAnnotator() {
     }
 
     /**
+     * Provides a way to query the Constant Value Checker, which computes the values of expressions
+     * known at compile time (constant propagation and folding).
+     */
+    ValueAnnotatedTypeFactory getValueAnnotatedTypeFactory() {
```

Comment:
The annotated type factory of a subchecker should not be cached.  See [the documentation of getTypeFactoryOfSubchecker](https://checkerframework.org/api/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.html#getTypeFactoryOfSubchecker-java.lang.Class-).